RestaurantManagementSystem/RestaurantManagementSystem/src/Core/Entities/RegistrationRequest.cs

```diff
@@ -20,7 +20,7 @@ public class RegistrationRequest
         [Required]
         public ContactInfo ContactInfo { get;  }
         public string VerificationCode { get; }
-        public RequestStatus.Status Status { get; private set; }
+        public Status Status { get; private set; }
```

Comment:
If you needed to use this `Status` class elsewhere the name will be ambiguous, better name would be more descriptive `RegistrationRequestStatus`